build.gradle

```diff
@@ -51,10 +51,10 @@ dependencies {
   compile 'org.ow2.asm:asm:5.0.4'
   compile 'org.ow2.asm:asm-commons:5.0.4'
   compile 'org.ow2.asm:asm-tree:5.0.4'
-  compile 'org.elasticsearch:elasticsearch:6.5.4'
+  compile 'org.elasticsearch:elasticsearch:6.6.0'
```

Comment:
lucene-expressions should be bumped to 7.5.1 because es 6.6.0 is compiled against lucene 7.5.1

Reply:
It doesn't look like 7.5.1 exists? https://mvnrepository.com/artifact/org.apache.lucene/lucene-expressions. Referencing https://raw.githubusercontent.com/elastic/elasticsearch/v6.6.0/buildSrc/version.properties as found from https://github.com/o19s/elasticsearch-learning-to-rank/issues/211#issuecomment-463125129, however, may imply 7.6? I've bumped it to 7.6 for now.

After building I get the errors
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':thirdPartyAudit'.
> Could not resolve all files for configuration ':runtime'.
   > Could not find org.apache.lucene:lucene-expressions:7.5.1.
     Searched in the following locations:
       - https://artifacts.elastic.co/maven/org/apache/lucene/lucene-expressions/7.5.1/lucene-expressions-7.5.1.pom
       - https://artifacts.elastic.co/maven/org/apache/lucene/lucene-expressions/7.5.1/lucene-expressions-7.5.1.jar
       - https://jcenter.bintray.com/org/apache/lucene/lucene-expressions/7.5.1/lucene-expressions-7.5.1.pom
       - https://jcenter.bintray.com/org/apache/lucene/lucene-expressions/7.5.1/lucene-expressions-7.5.1.jar
       - https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-expressions/7.5.1/lucene-expressions-7.5.1.pom
       - https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-expressions/7.5.1/lucene-expressions-7.5.1.jar
       - file:/home/ubuntu/.m2/repository/org/apache/lucene/lucene-expressions/7.5.1/lucene-expressions-7.5.1.pom
       - file:/home/ubuntu/.m2/repository/org/apache/lucene/lucene-expressions/7.5.1/lucene-expressions-7.5.1.jar
     Required by:
         project :
```

Reply:
indeed, sorry for the confusion!